pxtarget.json

```diff
@@ -88,7 +89,6 @@
         "libs/sparkfun-samd21-mini-breakout",
         "libs/sparkfun-redboard-turbo",
         "libs/sparkfun-lumidrive",
-        "libs/brain-nrf52840",
```

Comment:
did you forget to pull master?

Reply:
sorry what do you mean?

Reply:
I have not been able to test it with the device yet, I wanted to know if the files looked good before testing it